cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc_grpc.pb.go

```diff
@@ -15,6 +15,10 @@ limitations under the License.
 */
 
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+// versions:
```

Comment:
Everything in this file is auto-generated using 
```
protoc \
  -I ./cluster-autoscaler \
  -I ./cluster-autoscaler/vendor \
  --go_out=. \
  --go-grpc_out=. \
  ./cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc.proto
```
https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/externalgrpc#code-generation